tests/src/Kernel/SiteBrandingBlockTest.php

```diff
@@ -76,16 +83,53 @@ public function testBlockRendering(): void {
     $render = $this->container->get('renderer')->renderRoot($build);
     $crawler = new Crawler($render->__toString());
 
-    $actual = $crawler->filter('.site-name');
+    $actual = $crawler->filter('.bcl-header__site-name.site-name');
     $this->assertCount(1, $actual);
     $link = $actual->filter('.text-white.text-decoration-none.align-bottom');
     $this->assertCount(1, $link);
-    $actual = $crawler->filter('.site-logo');
+    $actual = $crawler->filter('.site-logo.d-none.d-lg-inline-block');
+    $this->assertCount(1, $actual);
+    $logo = $actual->filter('img');
+    $this->assertCount(1, $logo);
+    $expected = '/themes/custom/oe_whitelabel/logo.svg';
+    $this->assertSame($expected, $logo->attr('src'));
+
+    \Drupal::configFactory()->getEditable('oe_whitelabel.settings')
+      ->set('component_library', 'eu')
+      ->set('header_style', 'light')
+      ->save();
+    drupal_static_reset('theme_get_setting');
+
+    $builder = \Drupal::entityTypeManager()->getViewBuilder('block');
+    $build = $builder->view($entity, 'block');
+    $render = $this->container->get('renderer')->renderRoot($build);
+    $crawler = new Crawler($render->__toString());
+
+    $actual = $crawler->filter('.bcl-header__site-name.site-name');
+    $this->assertCount(1, $actual);
+    $link = $actual->filter('.text-dark.text-decoration-none.align-bottom');
+    $this->assertCount(1, $link);
+    $actual = $crawler->filter('.site-logo.d-none.d-lg-inline-block');
     $this->assertCount(1, $actual);
     $logo = $actual->filter('img');
     $this->assertCount(1, $logo);
     $expected = '/themes/custom/oe_whitelabel/logo.svg';
     $this->assertSame($expected, $logo->attr('src'));
+
+    \Drupal::configFactory()->getEditable('oe_whitelabel.settings')
+      ->set('component_library', 'neutral')
+      ->save();
+    drupal_static_reset('theme_get_setting');
+
+    $builder = \Drupal::entityTypeManager()->getViewBuilder('block');
+    $build = $builder->view($entity, 'block');
+    $render = $this->container->get('renderer')->renderRoot($build);
+    $crawler = new Crawler($render->__toString());
+
+    $actual = $crawler->filter('.bcl-header__site-name.site-name.h5.d-inline-block.d-lg-none');
+    $this->assertCount(1, $actual);
+    $link = $actual->filter('.text-dark.text-decoration-none.align-bottom');
+    $this->assertCount(1, $link);
```

Comment:
the test looks good to me, we test the branding structure and rendering, it's clean and clear, but, I have a suggestion, now that we have `CorporateHeaderLogosTest` we could rename it to `CorporateHeaderTest` and test the entire header section there with neutral, ec, eu.
It's just a suggestion, I don't insist, perhaps even discuss with @brummbar